src/org/omegat/core/data/RealProject.java

```diff
@@ -1814,7 +1814,7 @@ protected String getSegmentTranslation(String id, int segmentIndex, String segme
             this.config = props;
         }
 
-        Map<String, TMXEntry> data = new HashMap<>();
+        Map<String, ProjectTMXEntry> data = new HashMap<>();
```

Comment:
Let's change this to TreeMap, based on discussion https://github.com/omegat-org/omegat/commit/e740a9753201228a08048a3296377583ac2312bd